ADAL/src/public/ADTelemetry.h

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Please move this header into private for now until we're ready to ship the feature.